evap/static/ts/src/slider.ts

```diff
@@ -65,25 +62,25 @@ export class RangeSlider {
     public onRangeChange(): void {}
 
     public includeValues(values: number[]): void {
-        assert(Math.min(...values) >= this.min);
+        assert(Math.min(...values) >= this.allowed.low);
         const max = Math.max(...values);
-        if (max > this.max) {
-            this.max = max;
+        if (max > this.allowed.high) {
+            this.allowed.high = max;
             this.updateLimits();
             this.reset();
         }
     }
 
     public reset(): void {
-        this.range = { low: this.min, high: this.max };
+        this.value = { low: this.allowed.low, high: this.allowed.high };
     }
 
     private updateLimits(): void {
```

Comment:
Is this more fitting?
```suggestion
    private updateNestedElements(): void {
```
It might be misleading because we don't update the `value` attribute of the nested elements

Reply:
I just noticed, that I used it only once, so if the naming is misleading, it might be better to just inline this function into `includeValues`?

Reply:
hm, I think it's better with `updateNestedElements`